engine/src/flutter/impeller/base/mask.h

```diff
@@ -51,32 +51,9 @@ struct Mask {
 
   explicit constexpr operator bool() const { return !!mask_; }
 
-  // The following relational operators can be replaced with a defaulted
-  // spaceship operator post C++20.
+  // Comparison operators.
 
-  constexpr bool operator<(const Mask<EnumType>& other) const {
-    return mask_ < other.mask_;
-  }
-
-  constexpr bool operator>(const Mask<EnumType>& other) const {
-    return mask_ > other.mask_;
-  }
-
-  constexpr bool operator>=(const Mask<EnumType>& other) const {
-    return mask_ >= other.mask_;
-  }
-
-  constexpr bool operator<=(const Mask<EnumType>& other) const {
-    return mask_ <= other.mask_;
-  }
-
-  constexpr bool operator==(const Mask<EnumType>& other) const {
-    return mask_ == other.mask_;
-  }
-
-  constexpr bool operator!=(const Mask<EnumType>& other) const {
-    return mask_ != other.mask_;
-  }
+  constexpr auto operator<=>(const Mask<EnumType>& other) const = default;
```

Comment:
Do any operators other than == make sense for a mask? That assumes a lot about the values of the enum and how they relate.